src/handlers/__tests__/askingNotUsefulFeedback.test.js

```diff
@@ -33,51 +33,34 @@ let commonParamsNone = {
 it('handles "none" postback with no other existing feedbacks', async () => {
   gql.__push(apiResult.oneFeedback);
 
-  expect(await askingReplyFeedback(commonParamsNone)).toMatchSnapshot();
+  expect(await askingNotUsefulFeedback(commonParamsNone)).toMatchSnapshot();
 });
 
 it('handles "none" postback with other existing feedbacks', async () => {
   gql.__push(apiResult.twoFeedbacks);
 
-  expect(await askingReplyFeedback(commonParamsNone)).toMatchSnapshot();
+  expect(await askingNotUsefulFeedback(commonParamsNone)).toMatchSnapshot();
 });
 
-let commonParamsComment = {
-  data: {
-    searchedText: '貼圖',
-    foundArticleIds: [
-      'AWDZYXxAyCdS-nWhumlz',
-      '5483323992880-rumor',
-      'AV-Urc0jyCdS-nWhuity',
-      'AVsh8u7StKp96s659Dgq',
-    ],
-    selectedArticleId: 'AWDZYXxAyCdS-nWhumlz',
-    foundReplyIds: ['AWDZeeV0yCdS-nWhuml8'],
-    selectedReplyId: 'AWDZeeV0yCdS-nWhuml8',
-  },
-  state: 'ASKING_NOT_USEFUL_FEEDBACK',
-  event: {
-    type: 'text',
-    input: 'comment',
-    timestamp: 1519019734813,
-    postback: { data: '{"input":"comment","issuedAt":1519019701265}' },
-  },
-  issuedAt: 1519019735467,
-  userId: 'Uaddc74df8a3a176b901d9d648b0fc4fe',
-  replies: undefined,
-  isSkipUser: false,
+let commonParamsComment = JSON.parse(JSON.stringify(commonParamsNone));
```

Comment:
Let's use `const` here so that readers can know its's a constant.

Also for `let commonParamsNone = {` at the beginnin of this file.